lti_consumer/models.py

```diff
@@ -187,6 +188,18 @@ class LtiConfiguration(models.Model):
                   'Tool. One of either lti_1p3_tool_public_key or lti_1p3_tool_keyset_url must not be blank.'
     )
 
+    lti_1p3_redirect_uris = models.JSONField(
+        "LTI 1.3 Redirect URIs",
+        default=list,
+        blank=True,
+        # TODO: Further clarify this help
+        help_text="Valid urls the Tool may request us to redirect the id token to. The redirect uris "
```

Comment:
@MichaelRoytman 

Do you think this help text is more clear or helps illustrate the intent?